### ISSUE TITLE
Make check should ensure that we have a space between a period and a link.

### ISSUE THREAD

**Issue by ph** [elastic/beats#13865]
In the changelog (and elsewhere), there must be a space between a period and link, or the link will no longer render as a link (it will display in plain text).

Right: Some text. {pull}8177[8177]
Wrong: Some text.{pull}8177[8177]

**Comment by ph** [comment]
cc @mikemadden42 and @dedemorton

**Comment by ycombinator** [comment]
Would it make sense to update `make update` to insert the space, if it's absent?

**Comment by dedemorton** [comment]
We should not do this across the Beats documentation because there are legitimate instances where we have `.{`. For example, where we show commands for Windows that begin with `.{backslash}` and some of our multiline examples. I would restrict the test to checking for `.{pull}` in the changelog.

Will the check only run on changed files? There are still quite a few instances of missing spaces in previous versions. Since they all show up in our release notes, they might cause problems (not sure). I did not fix the problem in old changelogs because I figured it would complicate the merge from master to 7.x.

Not sure if we want to have automated changes (like inserting spaces) in the changelog, but as long as we restrict the changes to `.{pull}`, I think it would help...but we would end up with quite a few updates in old files (again, unless you constrain the update to new/changed files). IMO it would be good to fix the old versions if we can do that without messing up our changelog process.
